content/rancher/v2.x/en/installation/ha/_index.md

```diff
@@ -2,6 +2,78 @@
 title: Rancher HA Deployment
 weight: 2300
 draft: true
+version: v2.0
+lang: en
 ---
 
 ---
+
+**Introduction**
+
+For production deployments, Rancher should be deployed in HA mode. 
+
+Running Rancher HA mode means that rancher will be running as a k8s deployment in a k8s cluster. This provide Rancher deployment features like scale, upgrade, rollback... In this mode, access to rancher will be provided by k8s ingress controller. 
+
+Rancher HA could be deployed as addon in an exclusive RKE cluster or in an already deployed k8s cluster.
+
+Minimal of 3 nodes k8s cluster with overlapped planes is required.
+
+<img src="./rancher-HA-overlapped_planes.jpg" width="800" alt="Rancher HA diagram">
+
+**TLS**
+
+Rancher require tls for all inbound connections. It could be done directly, tls termination on Rancher, or by l7 lb, ingress, external lb's. It's needed that all clients (users, cli, cluster agents, node agents,...), trust on rancher tls certificates. There is no problem if tls certificates are signed by a well know public ca, but some considerations have to be done if a selfsigned ca is used.
+
+If selfsigned certificates are used, cacert needs to be provided to rancher. By default /etc/rancher/ssl/cacerts.pem . Rancher will use it to provide a ca-checksum to cluster/nodes agents to trust in it. Rancher autogenerate its own tls certificates and ca if they are not provided. 
+
+When Rancher is deployed in HA on a k8s/rke cluster, access to it is provided by ingress controller, a l7 load balancer. Ingress controller uses its own certificates by default. 
+
+Additionally, external lb's could be used to provide access to Rancher. External lb's uses its own certificates by default.
+
+Depending where tls termination is configured, 3 `tls-modes` could be used:
```

Comment:
Below are three scenarios for terminating TLS at different points of the network path:

### TLS termination at the Ingress
(...)
### TLS termination at the external LB
(...)
### TLS termination at the Rancher pod

Reply:
I think web presentation doesn't support this md format.